tests/InputFiles/GrMhd/GhValenciaDivClean/BinaryNeutronStar.yaml

```diff
@@ -287,6 +330,19 @@ EventsAndTriggersAtSlabs:
           - Name: PointwiseL2Norm(GaugeConstraint)
             NormType: L2Norm
             Components: Sum
+          - Name: PointwiseL2Norm(ThreeIndexConstraint)
```

Comment:
Want to also add the constraint energy? This trigger seems to now be run infrequently enough that it shouldn't affect performance